auction9-backend/serverless.yml

```diff
@@ -65,3 +65,12 @@ functions:
           path: myauctions
           method: get
           cors: true
+
+  # update status to 'inactive'
+  stopActiveAuction:
+    handler: handler.stopActiveAuction
+    events:
+      - http:
+          path: myauctions/{id}/stop
```

Comment:
ok, let's keep it like this for now, but consider having single PUT /auction/{id} endpoint which would handle any updates to an auction. of course, keeping in mind authorization aspect - only the owner can change the auction.